ComputeClient/Compute.Client.UnitTests/Server20/SnapshotAccessorTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DD.CBU.Compute.Api.Client;
+using DD.CBU.Compute.Api.Client.Server20;
+using DD.CBU.Compute.Api.Contracts.Requests;
+using DD.CBU.Compute.Api.Contracts.Requests.Snapshot;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Compute.Client.UnitTests.Server20
+{
+	[TestClass]
+	public class SnapshotAccessorTests : BaseApiClientTestFixture
+	{
+		[TestMethod]
+		public async Task GetSnapshotServicePlansPaginated_ReturnsResponse()
```

Comment:
Yeah its far from clear on the current naming convention. Sometimes its "Get" for List APIs and sometimes its a combination as in "List" for the base URL constant but "Get" in the methods/ classeses this URL constant is consumed from. Happy to go with either if we have a preference.

Reply:
My preference would be List across the board since that matches the API operation we are using too. The problem though is that if we move away from the inconsistency do we end up with code that becomes slightly less understandable.

Reply:
yeah 'List' is more consistent with our internal naming but its not widely used in the compute.api.client - 'Get....Paginated' seems to be convention in the project.

Reply:
@trevorflanagan have renamed the base method _ListSnapshotServicePlans_ - reads better for sure but wondering if it will lead to confusion as it doesn't seem to be the naming pattern used on alot of the existing APIs.

Reply:
Agreed it reads better to **me** and is less confusing for **me** but my confusion might not be as important as other people's confusion ;-)

I'm happy with going with what is consistent now and hold my nose.

Reply:
Check around the code in _IServerAccessor_ we do have _ListServersPaginated_ and _ListServers_ so probably have some precendence for going with _ListSnapshotServicePlans_ or even compromising on something like  _ListSnapshotServicePlansPaginated_

Reply:
If you are happy to go with what is here now, I am happy too.

Reply:
I'd be of a mind to use GetSnapshotServicePlansPaginated as it fits into the pattern established by the SnapshotAccessor. If the user of the ComputeAPI doesn't use the API directly then they'll have different confusions within the snapshot class, particularly if using intellisense. Realise I'm a little late to this party though :)